mne/time_frequency/tfr.py

```diff
@@ -380,6 +385,8 @@ def _compute_tfr(epoch_data, freqs, sfreq=1.0, method='morlet',
 
     if ('avg_' in output) or ('itc' in output):
         out = np.empty((n_chans, n_freqs, n_times), dtype)
+    elif output in ['complex', 'phase'] and n_tapers > 1:
```

Comment:
To me it makes more sense to always output the `tapers` dim, even if it's a singleton. It makes code and `.ndim` work the same regardless of if the time-bandwidth product / half-bandwidth (or whatever we call it) changes.

Reply:
yes, I agree, I'll change that